src/views/macros/loadComponentTemplate.njk

```diff
@@ -0,0 +1,3 @@
+{% macro loadComponentTemplate(componentName, params) %}
```

Comment:
Where does `params` get used?

Reply:
The parameters (`params`) are used inside each template file